README.md

```diff
@@ -82,36 +93,52 @@ Sitegeist:
       languageDimensionName: 'language'
 ```
 
+To enable automated translations for a language preset, just set `options.translationStrategy` to  `once` or `sync`.
+
+* `once` will translate the node only once when the editor switches the language in the backend while editing this node. This is useful if you want to get an initial translation, but work on the different variants on your own after that.
+* `sync` will translate and sync the node every time the node in the default language is published. Thus, it will not make sense to edit the node variant in an automatically translated language using this options, as your changed will be overwritten every time.
+
 If a preset of the language dimension uses a locale identifier that is not compatible with DeepL the deeplLanguage can
-be configured explicitly for this preset via `options.deeplLanguage`. If this value is set to null the language will neither
-be used as source nor as target for translations.  
+be configured explicitly for this preset via `options.deeplLanguage`.
 
-```
+```yaml
 Neos:
   ContentRepository:
     contentDimensions:
       'language':
         presets:
-        
-          # 
+
```

Comment:
Would love to see this in here to show the meaning of the `defaultPreset`

```
        #
        # The `defaultPreset` marks the source of for all tranaslations whith mode `sync`
        #  
        label: 'Language'
        default: 'en'
        defaultPreset: 'en'
        
        presets:
          
          #
          # English is the main language of the editors and spoken by editors,
          # the automatic translation is disabled therefore
          #
          'en':
            label: 'English'
            values: ['en']
            uriSegment: 'en'
            options:
              translationStrategy: 'none'

Reply:
Perfect!